src/main/resources/templates/views/admin/buildScansConfig.ftl

```diff
@@ -31,6 +31,7 @@
         [@ww.textfield labelKey="develocity.config.develocity-plugin.version" name="develocityPluginVersion"/]
         [@ww.textfield labelKey="develocity.config.ccud-plugin.version" name="ccudPluginVersion"/]
         [@ww.textfield labelKey="develocity.config.plugin-repository" name="pluginRepository"/]
+        [@ww.textfield labelKey="develocity.config.plugin-repository-credential-name" name="pluginRepositoryCredentialName"/]
```

Comment:
Is it possible to instead have a dropdown list with the list of available credentials?

Reply:
It's implemented now as we have it for the Access Key, but your question is still valid.
I'll play around with it and see if we can query the `CredentialsAccessor` and list them in UI somehow.

Reply:
It works.
I've introduced the default "None" value to the list, which sends empty text.
I also changed the "Shared Credential Name" field to be the same selector.

Reply:
Nice!